README.md

```diff
@@ -8,29 +8,21 @@ and facilitate access to data in Seeq.
 Seeq connectors can be written in Java or C# but this repository is intended to be used for developing C# Connectors. 
 C# development requires Windows operating system.
 
-It is recommended that you initially develop with a "private" version of Seeq Server that is unrelated to your
-production or official testing systems. You should install this private version directly on your development machine to
-minimize the amount of initial setup and troubleshooting.
-
-Install the version of Seeq Server that matches the version of the Connector SDK that you are using.
-
-If you need a license file for this purpose, contact [support@seeq.com](mailto:support@seeq.com).
-
 # Environment Setup
 
 ## The Build Environment
 
 The C# version of the SDK depends upon Microsoft Visual Studio for building and debugging. This SDK is tested with
-Microsoft Visual Studio 2015.
+Microsoft Visual Studio 2015 - 2019.
 
 The C# SDK also depends on .NET version 4.8. When upgrading .NET, you may need to restart your machine for the new
 version to take effect.
 
 To begin using the SDK:
 
 1. Launch the Windows Command Prompt.
-2. Change directory to the root directory of this SDK.
-3. At the prompt, execute the *environment script*: Type `environment`.
+1. Change directory to the root directory of this SDK.
+1. At the prompt, execute the *environment script*: Type `environment`.
```

Comment:
It was done to avoid reordering constantly. When it renders it monotonically increases:
![image](https://user-images.githubusercontent.com/5923087/217909341-f3bb2305-25a8-4a3d-87b2-b0c317b1b667.png)